ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -995,6 +1001,41 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
     self.hairline.alpha = alpha;
 }
 
+- (NSArray *)stepsForReviewStep:(ORKReviewStep *)reviewStep {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    if (reviewStep.isStandalone) {
+        for (ORKStep *step in reviewStep.steps) {
+            [steps addObject:step];
+        }
+    } else {
+        ORKStep *nextStep = reviewStep;
```

Comment:
Okay. So we should make `reverseListing` private. During initialization, `reverseListing` is set to `NO` for an embedded review step, and set to `YES` for a standalone review step. 

I like the idea of an `ORKOverviewStep`, which could be an embedded `ORKReviewStep` with reverseListing set to `YES`. If we want to avoid another method like `stepsForOverviewStep:` on the task view controller, we shouldn't use `_managedStepIdentifiers` here anyway.